### PR TITLE
DOC-4321: Reorganize all sections and develop Domain

### PR DESCRIPTION
**Pull request by hghorbel**

### CODE REVIEW

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Each domain represents a different context and specific applications to organize all rules and badges.
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
The following are the default domains that come with eXo Platform: 
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
.. note:: You can add, edit or remove domains using :ref: `Manage Domain <Manage Domain>` in Gamification Administration .
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Leaderboards display a snapshot of how your earned points compare to other users. It's a friendly competition designed to encourage everyone to earn more points. 
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
The position in the leaderboard is determined by the number of points earned over the period : Week, Month or All times and filtered by selected domain.
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
In the Leaderboard, the top 10 of users having earned most points, and every user is presented with their full name, profile picture and earned points.
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Your rank is displayed just below the top 10, if you are not in it. 
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
You can also load more 10 users by clicking on Load more button just below the Leaderboard.
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
When you mouse over a user in the leaderboard, a pie chart is displayed next to him to detail all proportions of points they earned in each domain.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
proofreading with substantial rewrite to enhance fluidity.

**Comment by hghorbel** [comment]
Update Done.

File: docs/Gamification.rst
Comment (mzorai) [inline docs/Gamification.rst]:
```suggestion
- Knowledge: Facts, information, and skills acquired through experiences using Forum, Wiki and Documents applications.
```

File: docs/Gamification.rst
Comment (mzorai) [inline docs/Gamification.rst]:
```suggestion
You can also load 10 more users by clicking on Load more button just below the Leaderboard.
```

**Review by mzorai — CHANGES_REQUESTED** [review]

**Review by mzorai — APPROVED** [review]

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
.. note:: You can add, edit or remove your own domains using :ref: `Manage Domain <Manage Domain>` in Gamification Administration .
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Leaderboards display a snapshot of users with most points. You always see your own rank so you can compare to others in various domains. It's a friendly competition designed to encourage everyone to engage in various activities by earning points. 
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
When you mouse over a user in the leaderboard, a pie chart is displayed beside with details of the distribution of points across domains. Hence you can discover what are their sweet spot domains and learn from their behaviours.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
More optimizations requested

**Comment by hghorbel** [comment]
Done.

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Leaderboards display a snapshot of users with most points. You always see your own rank so you can compare yourself to others in various domains. It's a friendly competition designed to encourage everyone to engage in various activities by earning points. 
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
The position in the leaderboard is determined by the number of points earned over the period : Week, Month or All times and is filtered by selected domain.
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
In the Leaderboard, the top 10 users with higher scores over the period are with their full name, profile picture and point scores.
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
You can also navigate in the ranks by clicking on Load more button to load 10 more entries.
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
When you mouse over a user in the leaderboard, a pie chart is displayed to illustrate the distribution of points across domains. Hence you can discover what are their sweet spot domains and learn from their behaviors.
```

**Review by plamarque — CHANGES_REQUESTED** [review]
2nd pass proofreading and suggested optimizations

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Domains are used to categorize all behaviors of the platform users. 
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
Each domain contains rules and badges related to a different context and usage of specific applications.
```

File: docs/Gamification.rst
Comment (plamarque) [inline docs/Gamification.rst]:
```suggestion
In the Leaderboard, the top 10 users with the highest scores over the period are displayed with their full name, profile picture and score (points gathered).
```

**Review by plamarque — CHANGES_REQUESTED** [review]
another pass for proofreading

**Comment by hghorbel** [comment]
Changes DONE.

**Comment by hghorbel** [comment]
@plamarque waiting for your approval to merge.

**Review by plamarque — APPROVED** [review]
LGTM

**Pull request merged**
